### PR TITLE
Cleanup jobs

### PR DESCRIPTION
**Pull request by volatilemolotov**
adds 
- beam_CleanUpGCPResources
- beam_CleanUpPrebuiltSDKImages

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: .github/workflows/beam_CleanUpGCPResources.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  security-events: read
+  statuses: read
+
+jobs:
+  beam_CleanUpGCPResources:
+    name: ${{matrix.job_name}} (${{matrix.job_phrase}}) 
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 120
+    strategy:
+      fail-fast: false
```
Comment (andreydevyatkin) [inline .github/workflows/beam_CleanUpGCPResources.yml:59]: I think this is redundant since there are no other jobs to run

Comment (volatilemolotov) [reply .github/workflows/beam_CleanUpGCPResources.yml]: Resolved. thanks

File: .github/workflows/beam_CleanUpGCPResources.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+# specific language governing permissions and limitations
+# under the License.
+
+name: Clean Up GCP Resources
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '* */6 * * *'
```
Comment (andreydevyatkin) [inline .github/workflows/beam_CleanUpGCPResources.yml:24]: It should be triggered once per day

File: .github/workflows/beam_CleanUpPrebuiltSDKImages.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+# specific language governing permissions and limitations
+# under the License.
+
+name: Clean Up Prebuilt SDK Images
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '0 */23 * * *'
```
Comment (andreydevyatkin) [inline .github/workflows/beam_CleanUpPrebuiltSDKImages.yml:24]: It should be triggered once per day. Should be something like this: `0 0 * * *`

File: .github/workflows/beam_CleanUpPrebuiltSDKImages.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  security-events: read
+  statuses: read
+
+jobs:
+  beam_CleanUpPrebuiltSDKImages:
+    name: ${{matrix.job_name}} (${{matrix.job_phrase}}) 
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 120
+    strategy:
+      fail-fast: false
```
Comment (andreydevyatkin) [inline .github/workflows/beam_CleanUpPrebuiltSDKImages.yml]: See the comment above

File: .github/workflows/beam_CleanUpPrebuiltSDKImages.yml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  pages: read
+  repository-projects: read
+  security-events: read
+  statuses: read
+
+jobs:
+  beam_CleanUpPrebuiltSDKImages:
+    name: ${{matrix.job_name}} (${{matrix.job_phrase}}) 
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 120
```
Comment (andreydevyatkin) [inline .github/workflows/beam_CleanUpPrebuiltSDKImages.yml:57]: See the comment above

File: .github/workflows/beam_CleanUpPrebuiltSDKImages.yml (hunk 3 above)
Comment (volatilemolotov) [reply .github/workflows/beam_CleanUpPrebuiltSDKImages.yml:24]: Done

File: .github/workflows/beam_CleanUpGCPResources.yml (hunk 2 above)
Comment (volatilemolotov) [reply .github/workflows/beam_CleanUpGCPResources.yml]: done

File: .github/workflows/beam_CleanUpGCPResources.yml [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  pages: read
+  repository-projects: read
+  security-events: read
+  statuses: read
+
+jobs:
+  beam_CleanUpGCPResources:
+    name: ${{matrix.job_name}} (${{matrix.job_phrase}}) 
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 120
```
[the remark this answers is not in the document]
Comment (volatilemolotov) [reply .github/workflows/beam_CleanUpGCPResources.yml]: done

File: .github/workflows/beam_CleanUpPrebuiltSDKImages.yml (hunk 4 above)
Comment (volatilemolotov) [reply .github/workflows/beam_CleanUpPrebuiltSDKImages.yml]: done

File: .github/workflows/beam_CleanUpPrebuiltSDKImages.yml (hunk 5 above)
Comment (volatilemolotov) [reply .github/workflows/beam_CleanUpPrebuiltSDKImages.yml]: Done. 

Thanks for taking a look

**Comment by volatilemolotov** [comment]
@damccorm @Abacn This one is ok but the workflows are failing at post setup:

https://github.com/apache/beam/actions/runs/6250265622/job/16969850271?pr=28528

Any idea what might be causing this?

**Comment by damccorm** [comment]
> @damccorm @Abacn This one is ok but the workflows are failing at post setup:
> 
> https://github.com/apache/beam/actions/runs/6250265622/job/16969850271?pr=28528
> 
> Any idea what might be causing this?

@volatilemolotov could you try pulling in the latest changes from master?

**Review by damccorm — APPROVED** [review]
LGTM once we get the workflows green (I think pulling in master should do it)

**Comment by volatilemolotov** [comment]
All green once master changes have been pulled in

**Comment by damccorm** [comment]
Thanks!

**Pull request merged**